src/lib/middleware-delete-cookie.ts

```diff
@@ -1,18 +0,0 @@
-import type { NextRequest, NextResponse } from 'next/server'
-
-/**
- * Copied from https://github.com/vercel/next.js/issues/40146#issuecomment-1236392823
- *
- * Works around a bug in Next.js middleware which does not delete a cookie when only response.cookies.delete is called.
- */
-export const deleteCookie = (
```

Comment:
Oh yay this was fixed??

Reply:
Yeah! As far as I can tell it has been. We also aren't deleting cookies in middleware anymore without the opt-in stuff

Reply:
Oh nvm I see we just weren't using it anymore?